### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by fin-github**
# Description
This adds a installer for it and runs the nessicary package commands. Eventually I'll make it so that it will react to error codes and lock the installer to py 3.10.9. If anyone wants to add on to this (For example allow config registration) it is grately appreciated!.

# Issue Fixes

If people are new to python they may have issues doing it and give up doing it because it may be hard to them. With this it automatically installs the required packages.

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have formatted and linted my code using python-black and pylint
- [ ] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

To test it simplily set your working directory to RedditVideoMakerBot-develop and run the command `py betterinstaller.py`. Then, follow the instructions.

### CODE REVIEW

**Comment by fin-github** [comment]
I will make a config writer in the installer soon!

File: betterinstaller.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+else:
+    print("Aborted.\nProgram will now quit.")
+    quit("Aborted")
+print("WARNING: What will happen soon is just it installing all of the python modules.\n It is not hacking your computer if you really think it is lol.")
+time.sleep(3)
+
+# Start installing python modules
+# Used ChatGPT to get code to use the OS module for this
+
+# Run a command to list the files in the current directory
```
Comment (fin-github) [inline betterinstaller.py:44]: Sorry! Forgot to take this comment off from ChatGPT!!

**Comment by callumio** [comment]
This doesn't seem particularly necessary. The current version already sets up the config. The only thing this really adds is a python script to install the dependencies - something that can already be done in ~2 commands by the user. If it were to be added into a script, a simple shell script would suffice (see the already existing `install.sh`)

Your 'universal clear' function is, by your own description, not universal since you say it wont clear tall consoles and adds pointless overhead to an otherwise simple script.

Something like
```python
import os
os.system('cls||clear')
``` 
is a far cleaner solution, or
```python
print('\x1b[2J')
```
also works on (nearly all) modern terminal emulators.

Additionally, there are spelling mistakes all over the place and far too many unnecessary sleep statements.

You've also checkmarked that you are pushing to the develop branch - you are not.

**Comment by fin-github** [comment]
Where are spelling mistakes??? Also sorry that I wasnt. I thought I chose that in the menu but I must have not.

**Pull request closed without merging**